en-GB.xml

```diff
@@ -0,0 +1,123 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<localisation>
+    <string id="bd490f4b1d73c0f8057256772b569c69">
+        <![CDATA[rBiblia - a free program for browsing Bible translations]]>
+    </string>
+    <string id="845b40f37ce2fb98e02aabfe5f8ab197">
+        <![CDATA[rBiblia - you way to read the Bible]]>
```

Comment:
Hello @ElanorFairbairn and thank you for your contribution :-)

I'm not sure about this line, shouldn't be `your way...`?